assets/blocks/quiz/question-block/question-edit.js

```diff
@@ -132,30 +132,19 @@ const QuestionEdit = ( props ) => {
 		]
 	);
 
-	const template = [
-		[
-			questionDescriptionBlock.name,
-			{},
-			[
-				[
-					'core/paragraph',
-					{
-						placeholder: __(
-							'Add question description or type / to choose a block.',
-							'sensei-lms'
-						),
-					},
-				],
-			],
```

Comment:
Seems like this template from the outer block was overriding the content coming from post content.